pybamm/discretisations/discretisation.py

```diff
@@ -431,7 +433,7 @@ def process_dict(self, var_eqn_dict):
 
             new_var_eqn_dict[eqn_key] = self.process_symbol(eqn)
 
-            new_var_eqn_dict[eqn_key].test_shape()
+            # new_var_eqn_dict[eqn_key].test_shape()
```

Comment:
can this be uncommented or removed entirely? will the shape of this be tested elsewhere?

Reply:
had removed because it was slow but I meant to uncomment